docs/index.rst

```diff
@@ -0,0 +1,29 @@
+TorchServe
+==================================
+
+TorchServe is a flexible and easy to use tool for serving PyTorch models.
+
+.. warning ::
+     TorchServe is experimental and subject to change.
+
+
+Basic Features
+----------------
+
+* :ref:`Serving Quick Start <install:Serve a model>` - Basic server usage tutorial
```

Comment:
Do we have to use `ref`? I'm not that familiar with this annotation, but it pretty much makes the index.rst file in the repo unusable from GitHub. Seems like it'll only work when rendered by Sphinx...

Reply:
Not a show stopper if we're going towards the docs only being used from the website. But if so, the tagline for the project should point to the web version of the docs. (and we probably should keep a README.md in the docs folder with the info on how the docs are built and a link to the most recent output of the docs.)